pkg/scheduler/api/devices/nvidia/vgpu/device_info.go

```diff
@@ -147,11 +165,24 @@ func (gs *GPUDevices) AddResource(pod *v1.Pod) {
 					gs.Device[index].UsedMem += uint(deviceused.Usedmem)
 					gs.Device[index].UsedNum++
 					gs.Device[index].UsedCore += uint(deviceused.Usedcores)
+					_, ok := gs.Device[index].PodMap[pod.Name]
```

Comment:
How about wrap a func to do metrics update operation?

Reply:
ok